cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyMetrics.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.detector;
+
+import com.linkedin.kafka.cruisecontrol.detector.notifier.AnomalyType;
+import java.util.Map;
+
+
+public class AnomalyMetrics {
+  private final Map<AnomalyType, Double> _meanTimeBetweenAnomalies;
+  private final double _meanTimeToStartFix;
+  private final long _numSelfHealingStarted;
+  private final long _ongoingAnomalyDurationMs;
```

Comment:
I feel that if an `ongoing` anomaly implies that it is `unfixed`. Moreover, the corresponding JavaDoc already provides further details.
```
@param ongoingAnomalyDurationMs The duration of the ongoing (unfixed/unfixable) anomaly if there is any, 0 otherwise.
```
Hence, for brevity, I would prefer to avoid the name change.